.openpublishing.redirection.json

```
@@ -1249,6 +1249,21 @@
       "redirect_url": "/dotnet/standard/linq/build-linq-xml-examples",
       "redirect_document_id": true
     },
+    {
+      "source_path": "docs/csharp/programming-guide/concepts/linq/creating-xml-trees-linq-to-xml-2.md",
+      "redirect_url": "/dotnet/standard/linq/create-xml-trees-csharp",
```

I don't think you need the language in the file name. Let's keep it simple unless you need to distinguish it from another article

```suggestion
      "redirect_url": "/dotnet/standard/linq/create-xml-trees",
```